ambry-store/src/test/java/com.github.ambry.store/StoreFindTokenTest.java

```diff
@@ -140,34 +140,25 @@ public void serDeTest() throws IOException {
       doSerDeTest(new StoreFindToken(), StoreFindToken.VERSION_0, StoreFindToken.VERSION_1, StoreFindToken.VERSION_2);
 
       // Journal based token
-      // incarnationId cannot be null for VERSION_2
-      doSerDeTest(new StoreFindToken(offset, sessionId, null, false), StoreFindToken.VERSION_0,
```

Comment:
because, the previous tests inherently tested this path. In other words, even though we create StoreFindToken with incarnationId, during serialization in version 0 and version 1 incarnation Id is not accounted for. Also, have added tests in constructionErrorCasesTest for null incarnationId.